src/tests/unit/connection.test.js

```diff
@@ -0,0 +1,80 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import config from 'config';
+import _ from 'lodash';
+import proxyquireModule from 'proxyquire';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+// Prevent call thru to original dependencies
+const proxyquire = proxyquireModule.noCallThru();
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.use(sinonChai);
+
+afterEach(() => { sinon.restore(); });
```

Comment:
Here, it doesn't matter since I only have one top-level `describe`, but I guess this ensures that `sinon.restore()` is always run, even if another top-level `describe` block were to be added.